### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by jeffmendoza**
Need to fix some imports and work around vendoring problems

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2234)

### CODE REVIEW

**Comment by janetkuo** [comment]
LGTM

**Comment by janetkuo** [comment]
GitHub says `This branch is out-of-date with the base branch`, though

**Comment by jeffmendoza** [comment]
Yeah, it's been doing that even if there are no merge conflicts =/

File: test/examples_test.go [hunk 1]
```diff
@@ -37,13 +40,10 @@ import (
...
 	expvalidation "k8s.io/kubernetes/pkg/apis/extensions/validation"
-	"k8s.io/kubernetes/pkg/capabilities"
 	"k8s.io/kubernetes/pkg/apis/policy"
 	policyvalidation "k8s.io/kubernetes/pkg/apis/policy/validation"
+	"k8s.io/kubernetes/pkg/capabilities"
 	"k8s.io/kubernetes/pkg/registry/batch/job"
-	"k8s.io/kubernetes/pkg/runtime"
-	"k8s.io/kubernetes/pkg/types"
-	"k8s.io/kubernetes/pkg/util/validation/field"
 	"k8s.io/kubernetes/pkg/util/yaml"
```
Comment (janetkuo) [inline test/examples_test.go]: This should probably changed to `k8s.io/apimachinery/pkg/util/yaml` also

**Pull request merged**